sdk/core/azure-core-tracing-opentelemetry/src/samples/PublishEventsTracingSample.md

```diff
@@ -81,31 +81,20 @@ public class Sample {
 
     private static void doClientWork() {
         EventHubProducerClient producer = new EventHubClientBuilder()
-            .connectionString(connectionString)
-            .buildProducerClient();
+            .connectionString(CONNECTION_STRING)
+            .buildProducer();
+
+        final int count = 2;
+        final byte[] body = "Hello World!".getBytes(UTF_8);
         
         Span span = TRACER.spanBuilder("user-parent-span").startSpan();
         try (final Scope scope = TRACER.withSpan(span)) {
-            final EventData event1 = new EventData("1".getBytes(UTF_8));
-            event1.addContext(PARENT_SPAN_KEY, span);
-    
-            final EventData event2 = new EventData("2".getBytes(UTF_8));
-            event2.addContext(PARENT_SPAN_KEY, span);
-
-            final List<EventData> telemetryEvents = Arrays.asList(event1, event2);
-            final CreateBatchOptions options = new CreateBatchOptions()
-                .setPartitionKey("telemetry")
-                .setMaximumSizeInBytes(256);
-    
-            EventDataBatch currentBatch = producer.createBatch(options);
-    
-            // For each telemetry event, we try to add it to the current batch.
-            for (EventData event : telemetryEvents) {
-                if (!currentBatch.tryAdd(event)) {
-                    producer.send(currentBatch);
-                    currentBatch = producer.createBatch(options);
-                }
-            }
+            final Context traceContext = new Context(PARENT_SPAN_KEY, span);
+            final Flux<EventData> testData = Flux.range(0, count).flatMap(number -> {
+                final EventData data = new EventData(body, traceContext);
+                return Flux.just(data);
+            });
+            producer.send(testData.toIterable(1));
```

Comment:
toIterable(1)? There is no exposed overload that allows you to send a single EventData in EventHubProducer.